CMakeLists.txt

```diff
@@ -265,6 +269,10 @@ endif()
 #   STB_IMAGE_INCLUDE_DIR - include directory for STB image library
 #   MATHJAX_ROOT - root of MathJax installation
 
+if (CMAKE_CROSSCOMPILING)
```

Comment:
It could be nice to add a comment here, even though it may seem obvious in the context of this PR.  Later maintainers (maybe even us if we forget about this months from now :smile:) might not know why this is needed.